BUILD.gn

```diff
@@ -301,6 +301,20 @@ static_library("electron_lib") {
     defines += [ "NODE_SHARED_MODE" ]
   }
 
+  if (is_win) {
```

Comment:
Should be moved to `chromium_src/BUILD.gn`

Reply:
done